machine-learning/tutorials/SentimentAnalysis/Program.cs

```diff
@@ -191,9 +193,11 @@ public static void UseLoadedModelWithBatchItems(MLContext mlContext)
 
             // <SnippetLoadModel>
             ITransformer loadedModel;
+            DataViewSchema dataViewSchema;
+
             using (var stream = new FileStream(_modelPath, FileMode.Open, FileAccess.Read, FileShare.Read))
```

Comment:
>using (var stream = [](start = 12, length = 19)

maybe add comment: load the model we saved previously

Reply:
We are moving load and save to it's own how-to.